### ISSUE TITLE
Update sdk to 27

### ISSUE THREAD

**Issue by RameshReddyKolla** [braintree/braintree-android-drop-in#58]
### General information
I am using SDK version 27 and also using this Braintree library in my project but showing warning two version (26 and 27)found. So can you please update the SDK version. Thank You.

**Comment by sdcoffey** [comment]
Hey @RameshReddyKolla,

Thanks for filing this report. Can you paste the content of the error messages you're seeing here? I'm having trouble reproducing this issue. Thanks!

**Comment by sdcoffey** [comment]
Hey @RameshReddyKolla,

I'm going to close this issue due to inactivity. Please open a new issue if you're still encountering an issue related to sdk version.

**Comment by sschendel** [comment]
This is still an issue as far as i can tell.  Please re-open...

gradle deps:
```
compile 'com.android.support:appcompat-v7:27.0.2'
compile 'com.braintreepayments.api:braintree:2.10.0'
compile 'com.braintreepayments.api:drop-in:3.2.1'
```


dependency tree:
```
+--- com.braintreepayments.api:drop-in:3.2.1
|    +--- com.braintreepayments.api:braintree:2.10.0 (*)
|    +--- com.braintreepayments:card-form:3.1.1
|    |    \--- com.android.support:design:26.0.0 -> 27.0.2 (*)
|    \--- com.android.support:cardview-v7:26.0.0
|         \--- com.android.support:support-annotations:26.0.0 -> 27.0.2
```

**Comment by sdcoffey** [comment]
@sschendel,

Unrelated, but if you're including Drop-in, you do not need to also include Braintree. Your dependency block should only include the support library and drop-in. As you can see, Drop-in will include Braintree as a dependency.

It looks like we need to update the supported versions of the support library in [Card form](https://github.com/braintree/android-card-form). We'll do that work, update the dependency in Drop-in and let you know when the releases are out.

**Comment by sschendel** [comment]
@sdcoffey good point on not needing `.api:braintree` dependency.  thanks.

**Comment by jmholtan** [comment]
I am also running into the support version issue. I am using the support version (27.0.2)  and when I try to exclude the support library from the drop-in ui sdk it causes the following error:

`Error:(55, 5) error: style attribute 'attr/textColorError (aka com.package.name:attr/textColorError)' not found.`

Hoping to see a fix shortly as we prefer to use the Drop-in UI.

Thanks!

**Comment by sdcoffey** [comment]
Hey @jmholtan,

I think I'm a little confused on why you're manually excluding the support library. Can you share a snippet of your build.gradle file? That error seems unrelated to the content of this issue.

**Comment by jmholtan** [comment]
Hey @sdcoffey,

I was excluding the support library on the braintree drop in ui via the below code because my project is using support version 27.0.2 whereas the braintree drop in ui sdk uses sdk 27 (related to the original **General Information** post above). I was receiving the redline in gradle stating there were multiple support versions.

Gradle Code to exclude support on BrainTree Drop In SDK:

`implementation('com.braintreepayments.api:drop-in:3.2.1'){
        ['com.android.support'].each { exclude group: it }
    }
`

However when I exclude the support library that is used in the DropIn SDK i get an error related to the attribute.

`Error:(55, 5) error: style attribute 'attr/textColorError (aka com.package.name:attr/textColorError)' not found.`

Because of this attribute error, we decided to use the non drop-in ui sdk and just roll our own payment UI. 

Thanks!

**Comment by NiRausz** [comment]
Hi, 

Am I the only one to still have the same error when trying to use `com.braintreepayments.api:drop-in:3.6.1`?
> error: style attribute 'attr/textColorError (aka au.com.stan.and.dev.debug:attr/textColorError)' not found

Wouldn't replacing textColorError by android:textColorError on your side fix the issue?
It would be a bit disappointing to have to use the non drop-in ui sdk (as jmholtan had to do) just because of a missing color attribute.
Thanks guys

**Comment by quinnjn** [comment]
@NiRausz the issue is that `textColorError` was renamed to `colorError`. 

There is an open PR to fix this issue #93, and you can watch the issue on #77. Note that there is a [workaround](https://github.com/braintree/braintree-android-drop-in/issues/77#issuecomment-403955038) until the PR is merged.
